client/blocks/term-form-dialog/index.jsx

```diff
@@ -211,12 +208,11 @@ class TermFormDialog extends Component {
 			errors.name = this.props.translate( 'Name required', { textOnly: true } );
 		}
 		const lowerCasedTermName = values.name.toLowerCase();
-		const matchingTerm = find( this.props.terms, ( term ) => {
-			return (
+		const matchingTerm = this.props.terms.find(
```

Comment:
`getTerms()` could return `null`. Should we add a safeguard and call `.find` with optional chaining?

Reply:
Correct, fixed in https://github.com/Automattic/wp-calypso/pull/57822/commits/eced1659fdefb0c12bd1020c5db6099819a7f2ae